src/components/profile/ProfileTeamLocation.vue

```diff
@@ -59,10 +57,36 @@ export default {
       return 'officeLocation:"' + this.officeLocation + '"'; // eslint-disable-line
     },
     timezoneWithTime() {
+      const currDate = new Date();
+      const dateOptions = {
+        year: 'numeric',
+        month: 'numeric',
+        day: 'numeric',
+        hour: 'numeric',
+        minute: 'numeric',
+        second: 'numeric',
+      };
+      if (!this.timezone || !this.currentTimezone) {
+        return null;
+      }
+      const profileDate = new Intl.DateTimeFormat('en-US', {
+        ...dateOptions,
+        timeZone: this.timezone,
+      }).format(currDate);
+      const currentLocalDate = new Intl.DateTimeFormat('en-US', {
+        ...dateOptions,
+        timeZone: this.currentTimezone,
+      }).format(currDate);
+      const hoursDiff =
+        (new Date(profileDate) - new Date(currentLocalDate)) / 36e5;
+      let printedOffset = '';
+      if (hoursDiff !== null && hoursDiff !== 0) {
+        printedOffset = ` | ${hoursDiff > 0 ? '+' : ''}${hoursDiff}hrs to you`;
```

Comment:
should we go with +/- here?

Reply:
We do. It just doesn't show the '-' here because if the hoursDiff is negative, the - comes with it.